Gemfile

```diff
@@ -19,6 +19,7 @@ gem 'htmlentities'
 gem 'us_states', git: 'git://github.com/GSA-OCSIT/us_states.git'
 gem 'git'
 gem 'rdiscount'
+gem 'sidekiq'
```

Comment:
You've added sidekiq to the Gemfile, but it seems you're not using it anywhere. In addition, in order to use sidekiq in staging/prod we need to make operational adjustments. Did you mean to add this here?